docs/glossary.md

```diff
@@ -10,45 +10,69 @@ tags:
 The following terms are used in [Temporal Platform](/concepts/what-is-the-temporal-platform) documentation.
 
 - [Activity](/concepts/what-is-an-activity)
+- [Activity Definition](/concepts/what-is-an-activity-definition)
 - [Activity Execution](/concepts/what-is-an-activity-execution)
-- [Activity Heartbeat](/concepts/what-is-an-activity-heartbeat)
+- [Activity Heartbeats](/concepts/what-is-an-activity-heartbeat)
```

Comment:
Only four terms in this list—Activity Heartbeats, Cron Jobs, Local Activities, and Signals—are plural; the rest are singular. Should those four terms also appear as singular?

Reply:
Yes, I believe so -